### DISCUSSION TITLE
Unable to verify

### DISCUSSION THREAD

**Discussion by SShattered** [community/community#132907] [GitHub Education] [answered]
### Hi everyone,

Hi,
I'm currently enrolled in BEng. (Hons) Electronic and Electrical Engineering programme on Liverpool John Moores university. This program is offered by a local university in my country. I'm getting problem with verifying, I'm trying to upload my transcript. My email is verified. When selected 'LJMU' as university, it doesn't let me upload the image, there's no upload button. Then I tried selecting my local university 'Sri Lanka Institute of Information Technology' then it shows upload button, but my transcript doesn't get verified. 

Please help me on this.

Also, I've requested from my University to provide me with a confirmation letter. Once I get it, I will try to upload it too.

**Comment by trivedikavya** [comment, marked as answer]
It sounds like you're trying to verify your student status through a platform that requires you to upload a transcript, but you're facing issues with the verification process.

Here are a few steps you can try:

1. Contact Support: Reach out to the support team of the platform you're trying to verify your student status on. Explain the issue in detail, including the fact that you are enrolled in a program from Liverpool John Moores University but through a local university in your country.

2. Clear Cache and Try Different Browser: Sometimes, issues with buttons not appearing can be due to browser problems. Clear your browser's cache and cookies, or try using a different browser to see if the upload button appears when you select LJMU.

3. Combine Documents: If you have both a transcript from LJMU and SLIIT, consider combining them into a single PDF file. This way, you can upload one document that includes proof from both institutions.

4. Wait for the Confirmation Letter: Once you receive the confirmation letter from your university, try uploading it. Sometimes a formal letter of enrollment or confirmation can help in verifying your status.

5. Check for Specific Instructions: Ensure you are following any specific instructions provided by the verification platform regarding document uploads. There might be certain formats or requirements that need to be met.

6. Temporary Solutions: As a temporary solution, upload the transcript under SLIIT and follow up with an email or support ticket to explain your situation. Provide any additional documentation they might request.

These steps should help you navigate the verification process. If you need further assistance, feel free to ask!

**Reply by SShattered** [reply]
Thank you for your response! I really appreciate it. As you suggested I opened a ticket and explained them the issue I have. I think it will take few days until a reply.

**Reply by trivedikavya** [reply]
can you make it as answer ...?
